tests/Classes/ListingTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+use ListingsApp\Classes\Listing;
+use PHPUnit\Framework\TestCase;
+require_once '../../src/Classes/Listing.php';
+
+
+class ListingTest extends TestCase{
+
+    public function testGetAgentRef_success()
```

Comment:
Sadly these tests are now failing - I believe Nate and Kash have a version of the Listing class with a constructor and a working tests file for this. Would be worth tracking them down and updating this test and Listing class to match